src/Constants.php

```diff
@@ -128,11 +128,11 @@ class Constants extends \SimpleSAML\XML\Constants
     /**
      * XML & XPath namespaces and identifiers
      */
-    public const XMLDSIGNS = 'http://www.w3.org/2000/09/xmldsig#';
-    public const XMLDSIG11NS = 'http://www.w3.org/2009/xmldsig11#';
+    public const NS_XDSIG = 'http://www.w3.org/2000/09/xmldsig#';
```

Comment:
Instead of renaming the constants, you should have probably renamed the cases where we use them..

Reply:
As discussed, we will start using new NS_ prefixed names.